package.json

```diff
@@ -69,7 +69,8 @@
   "scripts": {
     "test": "browserify test/index.js | tape-run | tap-spec",
     "validate": "jshint .",
```

Comment:
We could probably remove `validate` from `scripts` and `pre-commit` since the same thing is being covered by the `lint` script.

Alternatively we could make the `validate` script run `npm ls`. I think that's the new default for `precommit-hook@3` if no `validate` script exists.